src/databricks/labs/ucx/hive_metastore/federation.py

```diff
@@ -14,13 +18,26 @@
 )
 
 from databricks.labs.ucx.account.workspaces import WorkspaceInfo
+from databricks.labs.ucx.assessment.secrets import SecretsMixin
 from databricks.labs.ucx.config import WorkspaceConfig
 from databricks.labs.ucx.hive_metastore import ExternalLocations
 
 
 logger = logging.getLogger(__name__)
 
 
+@dataclass
+class ExtHms:
```

Comment:
I am not a fan of abbrevations, I understand the words can be come big, still by default write out the names and fallback on abbreviations by exception (or for the variable names, not the class names). For example, we have `ExternalLocations` and not `ExtLoc`.

Reply:
Fixed